src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -465,6 +465,7 @@ public void SetEncodingToNullThrows()
     }
 
     [Fact]
+    [Trait("Category", "SkipWhenLiveUnitTesting")]
```

Comment:
Maybe add a comment explaining that, so next GC pressure tests would follow the suit.